subcontracted_service/models/company.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# Author: Damien Crier
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api, fields
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    subcontracting_service_proc_rule_id = fields.Many2one(
+        comodel_name='procurement.rule',
+        string="Subcontracting_service procurement rule"
+    )
+
+    @api.multi
+    def _get_vals_for_proc_rule_subcontracting(self):
+        self.ensure_one()
+        warehouse = self.env['stock.warehouse'].search(
+            [('company_id', '=', self.id)],
+            limit=1
+        )
+        picking_type = self.env['stock.picking.type'].search(
+            [('code', '=', 'incoming'),
+             ('warehouse_id', '=', warehouse.id)
+             ],
+            limit=1
+        )
+        return {'name': 'Subcontracting service rule',
+                'company_id': self.id,
+                'action': 'buy',
+                'picking_type_id': picking_type.id,
+                }
+
+    @api.multi
+    def _set_subcontracting_service_proc_rule(self):
```

Comment:
The subcontracting service might need to be set up for every warehouse not for every company. 

Think in a company with several warehouses, if you sell from warehouse A some product (that is make to order) along with some also subcontracted service (which now happens to be defaulted with a rule that contains a picking type associated to warehouse B (the company's default). When both procurement are run, you will get this service attached to a different PO than the stockable product. That makes no sense when multiple warehouses sell make to order products with subcontracted services.